modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl

```diff
@@ -62,6 +62,11 @@ chmod -R ug+rwx ${forseti_home}/configs ${forseti_home}/rules ${forseti_home}/in
 echo "Installing Forseti"
 python3 setup.py install
 
+%{ if forseti_enable_tracing }
+echo "Installing Forseti tracing libs"
+python3 -m pip install --no-cache .[tracing]
+%{ endif }
+
 # Export variables required by initialize_forseti_services.sh.
```

Comment:
@red2k18 Please remove line 65-69 here, as tracing dependencies are now included in requirements.txt